### PR TITLE
Diverse translation improvements to sv.toml

### PR DESCRIPTION
**Pull request by migueldiez**
Changed some expressions to reflect more common language in Swedish web pages.

### CODE REVIEW

**Comment by migueldiez** [comment]
Note I added this one as a new pull request since I couldn't manage to add to the existing one (which has been stalled for some days).

suggested reviewers: @bittin , @xevz, @Millnert, @christiannageby, @ressonix  (from previous PR)

**Review by domingoruiz — APPROVED** [review]
The files are well assembled, the only thing left to check is that the translation is well done.

**Comment by migueldiez** [comment]
Well, I've been living in Sweden for more than 15 years now so I can defend myself a little bit, but we better see some reviews/enhancements from native speakers.  :-)

I think now is much more "standard professional Swedish", but I may have introduced some grammatical errors. 

Note: I would lend towards using "downstream vendor" vendor instead of "uppströmsleverantör" (it is very common to use English borrowed words in tech writing than the Swedish translation) as suggested in other comments. Maybe for some other commit, I do not have a strong opinion there.

File: i18n/sv.toml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  [faq.q]
+    [faq.q.downstream-partner-direction]
+    other = "Vad menar ni med att \"dess nedströmspartner har skiftat riktning?\""
+
+    [faq.q.come-in]
+    other = "Så var kommer Rocky Linux in??"
+
+  [faq.a]
+    [faq.a.downstream-partner-direction]
+    other = "CentOS projektet tillkännagav nyligen ett skifte i strategi för CentOS. Medan CentOS tidigare existerade som ett nedströmsbygge av dess uppströmsleverantör (det tar emot korrigeringar och uppdateringar efter att uppströmsleverantören gör det) kommer den att flyttas till att bara bli ett uppströmsbygge (testa korrigeringar och uppdateringar innan de inkluderas i uppströmsleverantören). Dessutom har stödet för CentOS Linux 8 förkortats, från 31 maj 2029 till 31 december 2021."
```
Comment (xevz) [inline i18n/sv.toml]:
```suggestion
    other = "CentOS-projektet tillkännagav nyligen ett skifte i strategi för CentOS. Medan CentOS tidigare existerade som ett nedströmsbygge av dess uppströmsleverantör (det tar emot korrigeringar och uppdateringar efter att uppströmsleverantören gör det) kommer den att flyttas till att bara bli ett uppströmsbygge (testa korrigeringar och uppdateringar innan de inkluderas i uppströmsleverantören). Dessutom har stödet för CentOS Linux 8 förkortats, från 31 maj 2029 till 31 december 2021."
```

File: i18n/sv.toml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  other = "Rocky Linux"
+
+  [landing.p1.content]
+  other = "En gemenskapsdriven insats för att ge dig en företagsklass, produktionsklar Linux."
+
+  [landing.p2.heading]
+  other = "Vad är Rocky Linux Projektet?"
+
+  [landing.p2.content]
+  other = "Rocky Linux är ett gemenskapsdrivet operativsystem utformat för att vara 100% bugg-för-bugg-kompatibelt med Amerikas främsta företagsklass Linux distribution nu när dess nedströms partner har skiftat inriktning. Det är under intensiv utveckling av gemenskapen. Rocky Linux leds av Gregory Kurtzer, grundare av CentOS-projektet. Det finns inget datum satt för första utgåvan. Bidragsgivare uppmanas att nå oss med hjälp av de kommunikationsalternativ som erbjuds på denna webbplats."
```
Comment (xevz) [inline i18n/sv.toml:65]:
```suggestion
  other = "Rocky Linux är ett gemenskapsdrivet operativsystem utformat för att vara 100% bugg-för-bugg-kompatibelt med Amerikas främsta Linuxdistribution i företagsklass nu när dess nedströmspartner har skiftat inriktning. Det är under intensiv utveckling av frivilliga. Rocky Linux leds av Gregory Kurtzer, grundare av CentOS-projektet. Det finns inget datum satt för första utgåvan. Bidragsgivare uppmanas att nå oss med hjälp av de kommunikationsalternativ som erbjuds på denna webbplats."
```

"gemenskapen" is not really a word used to replace the English word "community" in this context. Actually, I think maybe something like "en frivillig skara utvecklare" would be more idiomatic.

File: i18n/sv.toml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  [faq.title]
+  other = "Vanliga frågor"
+
+  [faq.q]
+    [faq.q.downstream-partner-direction]
+    other = "Vad menar ni med att \"dess nedströmspartner har skiftat riktning?\""
+
+    [faq.q.come-in]
+    other = "Så var kommer Rocky Linux in??"
```
Comment (xevz) [inline i18n/sv.toml]:
```suggestion
    other = "Så var kommer Rocky Linux in?"
```

File: i18n/sv.toml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+  [faq.a]
+    [faq.a.downstream-partner-direction]
+    other = "CentOS projektet tillkännagav nyligen ett skifte i strategi för CentOS. Medan CentOS tidigare existerade som ett nedströmsbygge av dess uppströmsleverantör (det tar emot korrigeringar och uppdateringar efter att uppströmsleverantören gör det) kommer den att flyttas till att bara bli ett uppströmsbygge (testa korrigeringar och uppdateringar innan de inkluderas i uppströmsleverantören). Dessutom har stödet för CentOS Linux 8 förkortats, från 31 maj 2029 till 31 december 2021."
+
+    [faq.a.come-in]
+    other = "Rocky Linux mål är att fungera som ett nedströmsbygge som CentOS har gjort tidigare, bygga utgåvor efter de har lagts till av uppströmsleverantören, inte före."
```
Comment (xevz) [inline i18n/sv.toml]:
```suggestion
    other = "Rocky Linux mål är att fungera som ett nedströmsbygge som CentOS har gjort tidigare, att bygga utgåvor efter de har lagts till av uppströmsleverantören, inte före."
```

File: i18n/sv.toml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+    [errors.404.explanation]
+    other = "Den sida du försökte nå kunde inte hittas. Var god försök igen."
+
+[landing]
+  [landing.p1.heading]
+  other = "Rocky Linux"
+
+  [landing.p1.content]
+  other = "En gemenskapsdriven insats för att ge dig en företagsklass, produktionsklar Linux."
```
Comment (xevz) [inline i18n/sv.toml]:
```suggestion
  other = "En gemenskapsdriven insats för att ge dig en produktionsklar Linux i företagsklass."
```

Once again, maybe even "En produktionsklar Linuxdistribution i företagsklass, utvecklad av frivilliga"?

**Review by xevz — COMMENTED** [review]
Various grammar changes to counter "särskrivningar" and other minor things.

File: i18n/sv.toml (hunk 2 above)
Comment (migueldiez) [reply i18n/sv.toml:65]: ok.

File: i18n/sv.toml (hunk 4 above)
Comment (migueldiez) [reply i18n/sv.toml]: tack

File: i18n/sv.toml (hunk 5 above)
Comment (migueldiez) [reply i18n/sv.toml]: thanks,

**Review by domingoruiz — COMMENTED** [review]
Ok good.

**Pull request merged**